internal/db/db.go

```diff
@@ -109,12 +109,16 @@ func (db *Db) executeQueriesAndPopulateChannel(queries []string, statementResult
 	}
 }
 
-func (db *Db) ExecuteAndPrintStatements(statementsString string, outF io.Writer, withoutHeader bool, printMode enums.PrintMode) error {
+func (db *Db) ExecuteAndPrintStatements(isCommand bool, statementsString string, outF io.Writer, withoutHeader bool, printMode enums.PrintMode) error {
```

Comment:
I'm unsure if the user should specify that this is a command but I can't think of a better way to solve this right now.

Reply:
We don't need this extra parameter